save-backend/build.gradle.kts

```diff
@@ -25,12 +25,8 @@ openApi {
     }
 }
 
-tasks.named("jar") {
-    mustRunAfter("forkedSpringBootRun", "generateOpenApiDocs")
-}
-
-tasks.named("inspectClassesForKotlinIC") {
-    mustRunAfter("forkedSpringBootRun", "generateOpenApiDocs")
+run {
+    tasks.findByName("inspectClassesForKotlinIC")?.mustRunAfter("forkedSpringBootRun")
```

Comment:
what with `generateOpenApiDocs`?

Reply:
`generateOpenApiDocs` depends on `forkedSpringBootRun`, so it's not required.
but actually, the dependency is not in proper way: `forkedSpringBootRun` should depend on `inspectClassesForKotlinIC`.

Also, it's not required anymore with new IC in kotlin which is enabled by default in 1.8.20.
But will leave the old workaround (by will make it's optional)